dask/dataframe/tests/test_dataframe.py

```diff
@@ -3615,6 +3615,16 @@ def test_dataframe_reductions_arithmetic(reduction):
     )
 
 
+def test_dataframe_mode():
+    data = [["Tom", 10, 7], ["Farahn", 14, 7], ["Julie", 14, 5], ["Nick", 10, 10]]
+
+    df = pd.DataFrame(data, columns=["Name", "Age", "Num"])
+    ddf = dd.from_pandas(df, npartitions=3)
+
+    assert_eq(ddf.mode(), df.mode())
+    assert_eq(ddf.Name.mode(), df.Name.mode())
+
```

Comment:
Can you add a test for an empty dataframe? `df = pd.DataFrame(columns=['a', 'b'])`.

Reply:
What's the failure with `check_index=True`? I also suspect it's not a problem but want to verify.

Reply:
Yeah, no worries, the error as reported by pytest is shown below.

```python3
E           AssertionError: DataFrame.index are different
E           
E           DataFrame.index classes are not equivalent
E           [left]:  Int64Index([], dtype='int64')
E           [right]: Index([], dtype='object')
```

Reply:
Thanks. I opened https://github.com/pandas-dev/pandas/issues/33321 in pandas. Can you add a reference to that as a comment so we can remove the `check_index=False` in the future?

Reply:
added